.github/workflows/exporter.yaml

```diff
@@ -0,0 +1,29 @@
+name: Python package
+
+on:
+  push:
+    branches: [ "main" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+
+    steps:
+    - uses: actions/checkout@v4
+    - name: Set up Python 3.11
+      uses: actions/setup-python@v3
+      with:
+        python-version: "3.11"
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
```

Comment:
Doing a poetry install is a good idea, but the pip upgrade & poetry install (as in, installing poetry) would just add more time and possibly make the build runs non(/less) deterministic.

Maybe use something similar to this https://github.com/BioImage-Archive/bia-integrator/blob/main/.github/workflows/core.yaml#L21 with a fixed python/poetry version instead?